tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -113,11 +115,26 @@ test.describe.serial(`AI Lab extension installation and verification`, () => {
       }
       await playExpect(errorTab, `Error Tab was present with stackTrace: ${stackTrace}`).not.toBeVisible();
     });
+
     test(`Verify AI Lab extension is installed`, async ({ runner, page, navigationBar }) => {
       [page, webview] = await handleWebview(runner, page, navigationBar);
       aiLabPage = new AILabPage(page, webview);
       await aiLabPage.navigationBar.waitForLoad();
     });
+
+    test(`Enable GPU support`, async ({ page, navigationBar }) => {
+      test.skip(!AI_LAB_TEST_GPU_SUPPORT, 'GPU support is not enabled');
```

Comment:
In this case you are actually not testing the change of gpu at all in this test.
This env. var should be used for the other test describes depending if we want to run with gpu or not.

Reply:
modified